bucket/logseq.json

```diff
@@ -1,12 +1,12 @@
 {
-    "version": "0.0.20",
+    "version": "0.1.2",
     "description": "A privacy-first platform for knowledge sharing and management",
     "homepage": "https://logseq.com",
     "license": "AGPL-3.0-only",
     "architecture": {
         "64bit": {
-            "url": "https://github.com/logseq/logseq/releases/download/0.0.20/logseq-win-x64-0.0.20.exe#/dl.7z",
-            "hash": "3e943e6886aa71614a856417a3b0843bceba56064a2e4e2059a6d952486cb7eb"
+            "url": "https://github.com/logseq/logseq/releases/download/0.1.2/logseq-win-x64-0.1.2.exe",
```

Comment:
Really? Did you even test this before you requested review?

Reply:
@Ash258 Sorry, I tested checkver and it automatically update the hash and also the link. I did not test install.